src/components/MostPopular/PopularItem/PopularItem.style.js

```diff
@@ -0,0 +1,32 @@
+"use client"
+
+import styled from "styled-components"
+
+export const Wrapper = styled.div`
+  text-align: left;
+  margin: ${({ theme }) => theme.spacings.small} auto;
+`
+export const Category = styled.div`
+  margin: ${({ theme }) => theme.spacings.medium} auto;
+`
+
+export const Title = styled.h2`
+  font-size: ${({ theme }) => theme.fontSizes.small};
+`
+
+export const MetaData = styled.div`
+  display: flex;
+  gap: ${({ theme }) => theme.spacings.small};
+  font-size: ${({ theme }) => theme.fontSizes.xsmall};
+  margin: 0 auto;
+`
+
+export const Author = styled.p`
```

Comment:
You are applying same styles for both Author and Date. I wonder if it would make sense to place these 2 inside a div and apply the style only to the div instead

Reply:
This styled has been unified to AuthorNDate style